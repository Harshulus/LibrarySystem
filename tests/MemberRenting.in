add member adam
add member bob 
add collection SampleCollection.csv 
rent 100000 111111
rent 100001 111111
rent 100002 111112
member books 100000
member books 100001
exit
