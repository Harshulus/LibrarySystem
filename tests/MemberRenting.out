user: Success.

user: Success.

user: 17 books successfully added.

user: Success.

user: Book is currently unavailable.

user: No such member in system.

user: To Kill a Mockingbird (Harper Lee)

user: Member is not currently renting.

user: Ending Library process.
